exonum/src/encoding/fields.rs

```diff
@@ -304,43 +304,82 @@ impl<'a> Field<'a> for DateTime<Utc> {
     }
 }
 
+const HEADER_SIZE: usize = 1;
+const BODY_SIZE: usize = mem::size_of::<Ipv6Addr>();
```

Comment:
`Ipv6Addr` is Rust structure which layout is unstable, so it's size probably can vary. Can you rewrite `BODY_SIZE` const to be just a 128 bit (or 16 bytes).